### ISSUE TITLE
The parameter `measurement_noise` does not work!

### ISSUE THREAD

**Issue by CuiiGen** [nasa/prog_algs#220]
The outputs are the same with the case where no measurement noise is applied in the model. The parameter `measurement_noise` does not work!

Here is my code (Python 3.10.4 64-bit):

```python
import matplotlib.pyplot as plot
from numpy import array, diag
from prog_models import LinearModel


class Descending(LinearModel):
    inputs = ['u']
    states = ['x']
    outputs = ['x']
    events = ['zero']
    A = array([[0]])
    B = array([[0]])
    E = array([[-1]])
    C = array([[1]])
    D = array([[0]])
    F = None
    default_parameters = {
        'x0': {
            'x': 10
        },
        'process_noise': 0,
        'measurement_noise': 5
    }

    def initialize(self, u=None, z=None):
        return self.StateContainer(self.default_parameters['x0'])

    def threshold_met(self, x):
        return {
            'zero': x['x'] <= 0
        }


def run_model():
    m = Descending()

    def future_loading(t, x=None):
        return m.InputContainer({'u': [1]})

    config = {
        'dt': 0.01,
        'save_freq': 0.01,
        'threshold_keys': 'zero'
    }
    simulation_result = m.simulate_to_threshold(future_loading, **config)

    states = simulation_result.states
    outputs = simulation_result.outputs
    inputs = simulation_result.inputs
    states.plot()
    outputs.plot()

    x0 = MultivariateNormalDist(['x'], [0], [[0.01]])
    kf = KalmanFilter(m, x0)
    x_est = []
    times = simulation_result.times
    for t, u, z in zip(times, inputs.data, outputs.data):
        kf.estimate(t, u, z)
        x_est.append(kf.x.mean)
    print(x_est)

    plot.show()


if __name__ == '__main__':
    run_model()

```

Without measurement nosie:

![image](https://user-images.githubusercontent.com/46126598/203540408-8eb4cf26-ec25-4af6-b849-4bae2d006b17.png)

With measurement nosie (`measurement_noise = 10`):

![image](https://user-images.githubusercontent.com/46126598/203540589-ddd1ce68-9875-4ec2-9eb3-81defae7a442.png)

**Comment by kjjarvis** [comment]
Hi @CuiiGen, thanks for bringing the measurement noise issue to our attention. We are working on a long-term fix, but in the meantime, you can include measurement noise by adding the following option to your config definition: 'print': True . Also please note that with this change, you'll either need to override event_state by redefining it in your script (see the linear_model.py example), or define F to be a matrix, for the code to run. 

We'll get another solution online next week that won't require printing. Thank you!

**Comment by teubert** [comment]
Measurement Noise wasn't working correctly in simulation when output was stored as a LazySimResult (e.g., when not printing to screen). This has been fixed in prog_models v1.4.4 - now released.

Thank you @CuiiGen for bringing this issue to our attention.
